docs/user_guide/index.rst

```diff
@@ -9,4 +9,6 @@ User Guide
 
    fairness_in_machine_learning
    assessment
-   mitigation
\ No newline at end of file
+   mitigation
+   further_reading
+   faq
```

Comment:
if you pull you'll see the "about" section (as a peer to user_guide) and you could add it there. Definitely a better place than the user guide. Additionally, you can try and add it also to contents.rst which would also add it at the top level. Not sure whether both of these things are possible. Otherwise just as a top-level item would also be fine.

Reply:
Probably still want to remove it from the user guide, right?

Reply:
Oops, yes.